tests/test_transform_api.py

```diff
@@ -0,0 +1,519 @@
+                        {'ok': 'Pu Si Lung22°37′38″B 102°47′09″Đ',
+                         'path': 2,
+                         'value': ['Pu Si Lung', '22°37′38″B 102°47′09″Đ']},
+                        {'ok': 'Kỷ Quan San (Bạch Mộc Lương Tử)22°30′28″B 103°35′15″Đ',
+                         'path': 3,
+                         'value': ['Kỷ Quan San (Bạch Mộc Lương Tử)', '22°30′28″B 103°35′15″Đ']},
+                        {'ok': 'Khang Su Văn22°45′12″B 103°26′24″Đ',
+                         'path': 4,
+                         'value': ['Khang Su Văn', '22°45′12″B 103°26′24″Đ']},
+                        {'ok': 'Tả Liên (Cổ Trâu)22°49′18″B 103°33′22.08″Đ',
+                         'path': 5,
+                         'value': ['Tả Liên (Cổ Trâu)', '22°49′18″B 103°33′22.08″Đ']},
+                        {'ok': 'Phú Lương (Tả Chì Nhù)',
+                         'path': 6,
+                         'value': ['Phú Lương (Tả Chì Nhù)', '']},
+                        {'ok': 'Nhìu Cô San', 'path': 7, 'value': ['Nhìu Cô San', '']},
+                        {'ok': 'Lùng Cúng', 'path': 8, 'value': ['Lùng Cúng', '']},
+                        {'ok': 'Nam Kang Ho Tao', 'path': 9, 'value': ['Nam Kang Ho Tao', '']},
+                        {'ok': 'Tà Xùa', 'path': 10, 'value': ['Tà Xùa', '']},
+                        {'ok': 'Lảo Thẩn', 'path': 11, 'value': ['Lảo Thẩn', '']},
+                        {'ok': 'Phu Xai Lai Leng19°12′B 104°11′Đ',
+                         'path': 12,
+                         'value': ['Phu Xai Lai Leng', '19°12′B 104°11′Đ']},
+                        {'ok': 'Ngọc Linh (Ngok Linh)',
+                         'path': 13,
+                         'value': ['Ngọc Linh (Ngok Linh)', '']},
+                        {'ok': 'Phu Tra', 'path': 14, 'value': ['Phu Tra', '']},
+                        {'ok': 'Tây Côn Lĩnh22°48\'26.0"B 104°47\'14.0"Đ',
+                         'path': 15,
+                         'value': ['Tây Côn Lĩnh', '22°48\'26.0"B 104°47\'14.0"Đ']},
+                        {'ok': 'Chư Yang Sin', 'path': 16, 'value': ['Chư Yang Sin', '']},
+                        {'ok': 'Kiều Liêu Ti', 'path': 17, 'value': ['Kiều Liêu Ti', '']},
+                        {'ok': 'Mường Hoong', 'path': 18, 'value': ['Mường Hoong', '']},
+                        {'ok': 'Rào Cỏ18°09′27″B 105°24′45″Đ',
+                         'path': 19,
+                         'value': ['Rào Cỏ', '18°09′27″B 105°24′45″Đ']},
+                        {'ok': 'Ngok Phan', 'path': 20, 'value': ['Ngok Phan', '']},
+                        {'ok': 'Ngok Lum Heo', 'path': 21, 'value': ['Ngok Lum Heo', '']},
+                        {'ok': 'Ngok Krinh', 'path': 22, 'value': ['Ngok Krinh', '']}]
+    response = resp.json
+    assert resp.status_code == 200
+    assert len(response) == len(transformed_data)
+    assert response == transformed_data
+
+
+def test_api_transform_compilation_error(client, example_db):
```

Comment:
add test case for restrictedpython doesn't allow getitem and your solution can handle it.

Reply:
Added the test case!